webapp/app.js

```diff
@@ -160,6 +161,17 @@ app.post('/output', jsonParser, (req, res, next) => {
   }
 })
 
+// return all output filenames
+app.post('/attributes', jsonParser, async (req, res, next) => {
```

Comment:
POST is not an appropriate method if you just want to GET data (this applies to the "/output" endpoint, too)

Reply:
fixed here 943d4714b310778e04708c5a776551752e06b442